.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  workflow_call:
+
+jobs:
+  unit-test:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```

Comment:
Does this not need to be 3.8 given our charmcraft.yaml uses 20.04?

Reply:
Yeh I need to fix the `charmcraft.yaml` file to 22.04 and then also update the CI, thanks for the comment.